Added cargo guidebook entry and updated guidebook tree to match

Timemaster99 opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->

Added a guidebook entry for cargo with a complimentary shuttle piloting guide. Assumes the reader has knowledge of controls and nothing else. Is comprehensive, and can be edited in the future to add tips if need be. Equipment section is copied from the wiki.

**Media**
<!-- 
If applicable, add screenshots or videos to showcase your PR. Small fixes/refactors are exempt, but all PRs which make ingame changes 
(adding clothing, items, new features, etc) must include ingame media or the PR will not be merged, in accordance with our PR guidelines.
This makes it much easier for us to merge PRs and find media for progress reports. If you include media in your pull request, we 
may potentially use it in the SS14 progress reports, with clear credit given.

Use screenshot software like Window's built in snipping tool, ShareX, Lightshot, or recording software like ShareX (gif), ScreenToGif, or Open Broadcaster Software (cross platform).
If you're unsure whether your PR will require media, ask a maintainer.

Check one of the boxes below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl:
- add: Added cargo guidebook entry.

![image](https://user-images.githubusercontent.com/57200767/226255027-7eda87c2-2f6e-4ae6-b83b-777cecf69ba7.png)
![image](https://user-images.githubusercontent.com/57200767/226255048-3da0627b-9b18-40da-b719-1d508f5e9d2f.png)

Timemaster99 committed:
Added cargo guidebook entry and updated guidebook tree to match

Torun-S commented on Resources/Server Info/Guidebook/Cargo.xml:
```diff
@@ -0,0 +1,69 @@
+  <Box>
+  <GuideEntityEmbed Entity="AppraisalTool"/>
+  </Box>
+  As a Quartermaster your equipment is very basic, though you do come with an [color=#a4885c]appraisal tool[/color] in your pocket. This appraisal tool allows you to scan items about the station and determine their value. This will allow you to determine some starting items to sell to raise your money.
```
Punctuation, first sentence.

Torun-S commented on Resources/Server Info/Guidebook/Cargo.xml:
```diff
@@ -0,0 +1,69 @@
+  <GuideEntityEmbed Entity="CrateSecgear"/>
+  </Box>
+  </Box>
+  When the shuttle is docked, it's time to unload. Drag the boxes out and make sure to deliver them to their respective department. Each box will have a label attached to it that can be removed by right clicking and selecting remove label (or leaving it on there if you are lazy, as most are).
```
Color "remove label" in beige.

Torun-S commented on Resources/Server Info/Guidebook/Cargo.xml:
What you can do is take all the text in parentheses and color it in darker gray. You can find the exact hex value elsewhere in the guidebook, like in cookbook.xml.

Torun-S reviewed (commented):
Will work, can polish.

Timemaster99 committed:
Updated text coloring and punctuation.

- Colored "remove label" in beige.
- Colored text enclosed in parenthesis to dark gray.
- Fixed punctuation in the first sentence of the Equipment section.

Torun-S reviewed (approved):
Cool! Thanks for putting in the work!
